enterprise/views.py

```diff
@@ -7,6 +7,7 @@
 import re
 from logging import getLogger
 from uuid import UUID
+import six
```

Comment:
@rehanedly  I think we should not use six now since we have been already on python 3.

Reply:
fixed